electron/main.ts

```diff
@@ -16,11 +19,15 @@ import {
   readFileSync,
   unlinkSync,
   writeFile,
-  writeFileSync,
+  writeFileSync
 } from 'graceful-fs'
 import i18next from 'i18next'
 import Backend from 'i18next-fs-backend'
-import { cpus, userInfo as user } from 'os'
+import isOnline from 'is-online'
```

Comment:
I saw some discussion about this in Discord, so not sure if this was OK-ed to add this dependency.

Reply:
I think is ok to use this dependency because the built-in solution is a lot more complex since is based on a Event, so is more useful when a change on the network status happens. With this package, it will check the connectivity every time it is called. So for our use case I think is better.

Reply:
I'm marking this as resolved then.